src/platform_impl/linux/taskbar/unity.rs

```diff
@@ -0,0 +1,131 @@
+use zbus::{blocking::Connection, MessageBuilder};
+
+pub struct Manager {
+  conn: Connection,
+  app_uri: String,
+  count: i32,
+  progress: f64,
+  progress_visible: bool,
+  count_visible: bool,
+  urgent: bool,
+  dirty_count: bool,
+  dirty_progress: bool,
+  dirty_progress_visible: bool,
+  dirty_count_visible: bool,
+  dirty_urgent: bool,
+}
+
+enum DataTypes<'a> {
+  Str(&'a str),
+  String(&'a String),
+  Bool(&'a bool),
+  Number(&'a i32),
+  Float(&'a f64),
```

Comment:
No need for `Str` and `String` at the same time, also no need to take a reference for `bool`, `i32` and `f64` because they all live on the stack and derive `Copy`
```suggestion
  Str(&'a str),
  Bool(bool),
  Number(i32),
  Float(f64),
```

Reply:
But the compiler was just complaining about that (when I was debugging in kali-wsl)

Reply:
If you have a reference to a `Copy` value, you can just dereference it:
```rs
let x = 21; // x == 21
let y = &x; // y == &21
let d = *y; // d == 21
```

Reply:
Referencing as `*` is safe?

Reply:
Already Resolved